### PR TITLE
Fix #84: Update softwareupdate verbose flag

### LINKED ISSUE #84: softwareupdate: invalid option -- v
Hi.
In [osxprep.sh#L13](https://github.com/donnemartin/dev-setup/blob/master/osxprep.sh#L13]),
```
sudo softwareupdate -iva
```

But I found,
```
$ sudo softwareupdate -v
softwareupdate: invalid option -- v
$ defaults read loginwindow SystemVersionStampAsString
10.12.6
```

Same error occured on `10.13.3`.

We might need detect osx version and switch options, or simply delete `-v`.

I don't have other osx so I cannot confirm from which version this error occurs.

### PR DESCRIPTION
**Pull request by allanice001**
For OSX Mojave:
$ sudo softwareupdate -iva
Password:
softwareupdate: invalid option -- v
usage: softwareupdate <cmd> [<args> ...]

** Manage Updates:
	-l | --list		List all appropriate update labels (options:  --no-scan, --product-types)
	-d | --download		Download Only
	-e | --cancel-download		Cancel a download
	-i | --install		Install
		<label> ...	specific updates
		-a | --all		All appropriate updates
		-R | --restart		Automatically restart (or shut down) if required to complete installation.
		-r | --recommended	Only recommended updates
	--background		Trigger a background scan and update operation
	--ignore <label> ...	Ignore specific updates
	--reset-ignored		Clear all ignored updates

** Other Tools:
	--dump-state		Log the internal state of the SU daemon to /var/log/install.log
	--evaluate-products	Evaluate a list of product keys specified by the --products option 
	--history		Show the install history.  By default, only displays updates installed by softwareupdate.  
	--all 			Include all processes in history (including App installs) 

** Options:
	--no-scan		Do not scan when listing or installing updates (use available updates previously scanned)
	--product-types <type>		Limit a scan to a particular product type only - ignoring all others
		Ex:  --product-types macOS  || --product-types macOS,Safari 
	--products		A comma-separated (no spaces) list of product keys to operate on. 
	--force			Force an operation to complete.  Use with --background to trigger a background scan regardless of "Automatically check" pref 

	--verbose		Enable verbose output
	--help			Print this help

### CODE REVIEW

**Comment by johnwyles** [comment]
This is valid

File: osxprep.sh [hunk 1]
```diff
@@ -10,11 +10,11 @@ while true; do sudo -n true; sleep 60; kill -0 "$$" || exit; done 2>/dev/null &
 echo "------------------------------"
 echo "Updating OSX.  If this requires a restart, run the script again."
 # Install all available updates
-sudo softwareupdate -iva
+sudo softwareupdate -ia
```
Comment (kirk-clover) [inline osxprep.sh]:
```suggestion
sudo softwareupdate -ia --verbose
```

File: osxprep.sh [hunk 2]
```diff
@@ -10,11 +10,11 @@ while true; do sudo -n true; sleep 60; kill -0 "$$" || exit; done 2>/dev/null &
...
 # Install only recommended available updates
 #sudo softwareupdate -irv
```
Comment (kirk-clover) [inline osxprep.sh]:
```suggestion
#sudo softwareupdate -ir --verbose
```

**Comment by donnemartin** [comment]
@kirk-clover appreciate the suggested changes, unfortunately the UI isn't allowing me to apply them.  I'll apply manually.

**Comment by donnemartin** [comment]
@allanice001 @kirk-clover @RupGautam thank you!

**Pull request merged**